shared/src/main/java/eu/darkbot/shared/modules/CollectorModule.java

```diff
@@ -54,6 +55,21 @@
 
     private long invisibleUntil, waitingUntil;
 
+    public CollectorModule(PluginAPI api) {
+        this(api.requireAPI(BotAPI.class),
+                api.requireAPI(PetAPI.class),
+                api.requireAPI(HeroAPI.class),
+                api.requireAPI(StarSystemAPI.class),
+                api.requireAPI(StatsAPI.class),
+                api.requireAPI(ConfigAPI.class),
+                api,
```

Comment:
I'd say as a general rule of thumb, if PluginAPI is required pass it always as first parameter, it's a bit unnatural to have it mid-way